vendor/src/github.com/docker/engine-api/types/container/config.go

```diff
@@ -5,6 +5,20 @@ import (
 	"github.com/docker/go-connections/nat"
 )
 
+// HealthConfig holds configuration settings for the HEALTHCHECK feature.
+// For durations, a value of -1 means to inherit the default value.
```

Comment:
At the request of @stevvooe I've removed the pointers again. Now that `grace` isn't an option, 0 is no longer a valid duration for any option and we can use zero for inherit.